### ISSUE TITLE
HTML parser: crashes if table described in one line

### ISSUE THREAD

**Issue by braska** [remarkjs/react-markdown#259]
I've opened https://rexxars.github.io/react-markdown/ and tried to check following snippet and all works fine:
```html
<table>
  <tr>
    <td>1</td>
    <td>a</td>
  </tr>
</table>
```

But if I am trying to use this one all crashes:
```html
<table><tr><td>1</td><td>a</td></tr></table>
```
Stack trace from console:
```
Uncaught TypeError: (el.props.children || []).concat is not a function
    at mergeNodeChildren (ast-to-react.js?5e65:214)
    at getNodeProps (ast-to-react.js?5e65:175)
    at astToReact (ast-to-react.js?5e65:18)
    at eval (ast-to-react.js?5e65:23)
    at Array.map (<anonymous>)
    at resolveChildren (ast-to-react.js?5e65:22)
    at astToReact (ast-to-react.js?5e65:19)
    at ReactMarkdown (react-markdown.js?2ced:58)
    at ReactMarkdownWithHtml (with-html.js?7ca6:13)
    at ProxyFacade (react-hot-loader.development.js?9cb3:705)
```

**Comment by nlucero** [comment]
Seems to be happening with any one-line `HTML` string

**Comment by plainnany** [comment]
same question

**Comment by te-online** [comment]
I can confirm that it throws that error with one-line `HTML` strings.
Adding a newline fixes it temporarily. *Example:*
```Javascript
<ReactMarkdown source={`${content}\r\n `} escapeHtml={false} astPlugins={[parseHtml]} />
```

**Comment by ChristianMurphy** [comment]
this will be resolved in #428

**Comment by wooorm** [comment]
This should be solved by landing GH-563 today, which will soon be released in v6.0.0!
